### DISCUSSION TITLE
ScriptReference/Application-installerName

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#593] [Notes]
# ScriptReference/Application-installerName

https://docs.unity3d.com/ScriptReference/Application-installerName.html

**Comment by abrusle** [comment]
When installed via the Google Play Store this returns `com.google.android.packageinstaller`

**Reply by abrusle** [reply]
can also be `com.android.vending` or `com.android.chrome`... seems to vary depending on device, operating system and installer :/
